### PR TITLE
PLAT-120846 - Document mobile to web id sharing feature - WIP

### PR DESCRIPTION
**Pull request by calebk1**
https://jira.corp.adobe.com/browse/PLAT-120846?filter=-1

### CODE REVIEW

File: foundation-extensions/identity-for-edge-network/api-reference.md [hunk 1]
```diff
@@ -198,6 +198,125 @@ Identity.getIdentities { (identityMap, error) in
...
+
+### Java
+
+**Syntax**
+
+```java
+public static void getUrlVariables(final AdobeCallback<String> callback);
+```
+
+- `callback` has an NSString value that contains the visitor identifiers as a query string after the service request is complete.
```
Comment (emdobrin) [inline foundation-extensions/identity-for-edge-network/api-reference.md:232]: for java we should use `has a String value`

File: foundation-extensions/identity-for-edge-network/api-reference.md [hunk 2]
```diff
@@ -198,6 +198,125 @@ Identity.getIdentities { (identityMap, error) in
...
+
+    guard let urlWithVisitorData: URL = URL(string: urlStringWithVisitorData) else {
+      // handle error, unable to construct URL
+      return
+    }
+
+    // handle the retrieved urlVariables encoded string here
+    // APIs which update the UI must be called from main thread
+    DispatchQueue.main.async {
+    self.webView.load(URLRequest(url: urlWithVisitorData))
```
Comment (emdobrin) [inline foundation-extensions/identity-for-edge-network/api-reference.md:283]: formatting: this should go a tab inside.

**Review by emdobrin — COMMENTED** [review]
@calebk1 can you also update the main [release-notes page](https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/blob/master/release-notes/README.md) with the same?

**Review by emdobrin — APPROVED** [review]

File: release-notes/README.md [hunk 3]
```diff
@@ -6,6 +6,10 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 ## June 7, 2022
 
+### iOS & Android Identity 1.1.0
```
Comment (addb) [inline release-notes/README.md:9]: Let's break this into two different labels to easily search through release notes.

iOS AEPEdgeIdentity 1.1.0
Android EdgeIdentity 1.1.0

File: foundation-extensions/identity-for-edge-network/release-notes-1.md [hunk 4]
```diff
@@ -1,10 +1,16 @@
 # Release Notes
 
+## June 7, 2022
+
+### iOS & Android Identity 1.1.0
```
[the remark this answers is not in the document]
Comment (addb) [reply foundation-extensions/identity-for-edge-network/release-notes-1.md:5]: For ios release we need to add an additional entry for release notes along with getUrlVariables api.

[the remark this answers is not in the document]
Comment (addb) [reply foundation-extensions/identity-for-edge-network/release-notes-1.md:5]: So the release notes for Android will be: 

```
- Added the getUrlVariable API to support passing the visitor ID from a mobile app to a web view.
- Internal fixes for identityMap deserialization
```

**Pull request merged**